tests/tests/cpld_bmarkfrac_v16_nsst

```diff
@@ -0,0 +1,150 @@
+                   RESTART/phy_data.tile6.nc \
+                   RESTART/sfc_data.tile1.nc \
+                   RESTART/sfc_data.tile2.nc \
+                   RESTART/sfc_data.tile3.nc \
+                   RESTART/sfc_data.tile4.nc \
+                   RESTART/sfc_data.tile5.nc \
+                   RESTART/sfc_data.tile6.nc \
+                   RESTART/MOM.res.nc \
+                   RESTART/MOM.res_1.nc \
+                   RESTART/MOM.res_2.nc \
+                   RESTART/MOM.res_3.nc \
+                   RESTART/iced.2013-04-01-43200.nc \
+                   RESTART/ufs.cpld.cpl.r.2013-04-01-43200.nc"
+
+export_fv3
+export_cpl
+
+export SYEAR="2013"
+export SMONTH="04"
+export SDAY="01"
+export SHOUR="00"
+
+export DAYS="0.5"
+export FHMAX="12"
+export RESTART_INTERVAL="6"
+export RESTART_N=${RESTART_INTERVAL}
+
+export TASKS=$TASKS_cpl_bmrk
+export TPN=$TPN_cpl_bmrk
+export INPES=$INPES_cpl_bmrk
+export JNPES=$JNPES_cpl_bmrk
+export THRD=$THRD_cpl_bmrk
+export WRTTASK_PER_GROUP=$WPG_cpl_bmrk
+
+export med_petlist_bounds=$MPB_cpl_bmrk
+export atm_petlist_bounds=$APB_cpl_bmrk
+export ocn_petlist_bounds=$OPB_cpl_bmrk
+export ice_petlist_bounds=$IPB_cpl_bmrk
+
+# atm/ocn/ice resolution
+export ATMRES='C384'
+export NPX=385
+export NPY=385
+export IMO=1536
+export JMO=768
+
+export OCNRES='025'
+export ICERES='0.25'
+export NX_GLB=1440
+export NY_GLB=1080
+export NPROC_ICE='48'
+
+# resolution dependent setting
+export CDMBWD=${CDMBWD_c384}
+
+# set component and coupling timesteps
+export DT_ATMOS='225'
+export DT_CICE=${DT_ATMOS}
+export DT_DYNAM_MOM6='900'
+export DT_THERM_MOM6='1800'
+export CPL_SLOW=${DT_THERM_MOM6}
+export CPL_FAST=${DT_ATMOS}
+
+# nems.configure
+export coupling_interval_slow_sec=${CPL_SLOW}
+export coupling_interval_fast_sec=${CPL_FAST}
+
+export FRAC_GRID='.T.'
+export CPLMODE='nems_frac'
+
+export NPZ=127
+export NPZP=128
+export PSM_BC=1
+export DDDMP=0.2
+export NSTF_NAME=2,1,0,0,0
+
+# resolution dependent files
+export MOM_INPUT="MOM_input_template_${OCNRES}"
+export MESHOCN_ICE="mesh.mx${OCNRES}.nc"
+export CICEGRID="grid_cice_NEMS_mx${OCNRES}.nc"
+export CICEMASK="kmtu_cice_NEMS_mx${OCNRES}.nc"
+export CHLCLIM="seawifs-clim-1997-2010.${NX_GLB}x${NY_GLB}.v20180328.nc"
+export FRUNOFF="runoff.daitren.clim.${NX_GLB}x${NY_GLB}.v20180328.nc"
+
+export FNALBC="'global_snowfree_albedo.bosu.t766.1536.768.rg.grb',"
+export FNVETC="'global_vegtype.igbp.t766.1536.768.rg.grb',"
+export FNSOTC="'global_soiltype.statsgo.t766.1536.768.rg.grb',"
+export FNSMCC="'global_soilmgldas.statsgo.t766.1536.768.grb',"
+export FNABSC="'global_mxsnoalb.uariz.t766.1536.768.rg.grb',"
+
+export OZ_PHYS_NEW=".T."
+
+export MOM6_RIVER_RUNOFF='True'
+export MOM6_RESTART_SETTING="r"
+export MOM6_REPRO_LA='True'
+
+export RUNID="cpcice"
+
+export INPUT_NML=input.benchmark_v16.nml.IN
+
+export FIELD_TABLE="field_table_gfsv16"
+export DIAG_TABLE="diag_table_bmark_v16_template"
+export SUITE_NAME="FV3_GFS_v16_couplednsst"
```

Comment:
but shouldn't this be `export CCPP_SUITE="FV3_GFS_v16_couplednsst"` ? Please check other regression tests.

Reply:
@climbfuji The parm/input.benchmark_v16.nml.IN uses SUITE_NAME:

```
&atmos_model_nml
  blocksize = 32
  chksum_debug = .false.
  dycore_only = .false.
  fdiag = 6
  fhmax = 840
  fhout = 6
  fhmaxhf = 0
  fhouthf = -1
  ccpp_suite = '@[SUITE_NAME]'

/

Reply:
This inconsistency should be fixed at some point. It's unintuitive that `SUITE_NAME` environment variable is used to set the value of `ccpp_suite` namelist variable.

Reply:
Noted.

Reply:
This should be fixed after creating the baseline and before verifying against it. Many workflows depend on this being called `CCPP_SUITE` and not `SUITE_NAME`.

Reply:
The variable is ``SUITE_NAME`` in all the coupled tests, not just the v16 input nml. It requires a change in default_vars, the three input nmls that the coupled tests use( input.benchmark_ccpp.nml.IN, input.benchmark_v16.nml.IN, input.mom6_ccpp.nml.IN) and then in all the cpld tests which are using a non-default setting.

Reply:
Ok, this should never have been merged, but apparently I didn't catch it. Let's fix this in a separate PR then.

Reply:
I will add it to the CMEPS PR if you like.

Reply:
That would be great, yes. Thanks!